### ISSUE TITLE
Basic backend service

### ISSUE THREAD

**Issue by Zwirek009** [Zwirek009/RSOProject#3]
Create basic backend microservice. It should have REST api and be able to:
- response frontend service (#2)

**Comment by msypetkowski** [comment]
Example add beer scenario:
```
curl -X POST -c cookies1 'http://localhost:3000/api/sessions' -d'username=user&password=user'
curl -X GET -b cookies1 -c cookies1 'http://localhost:3000/api/users/current'
curl -X POST -b cookies1 -c cookies1 'http://localhost:3000/api/beer/add' -d'name=name1&beerType=ALE&desc=desc1&city=city1&district=disc'
```

**Comment by msypetkowski** [comment]
Following works:
`curl -X POST 'http://localhost:8082/beer/add' -d 'userId=1&name=name1&style=style1&abv=1&blg=2&ibu=3&date=4&left=5&price=6&desc=desc1&regionId=1'`

For following commands:
```

curl -X POST -c cookies1 'http://localhost:3000/api/sessions' -d'username=user&password=user'
curl -X GET -b cookies1 -c cookies1 'http://localhost:3000/api/users/current'
curl -X POST -b cookies1 -c cookies1 'http://localhost:3000/api/beer/add' -d 'name=name1&style=style1&abv=1&blg=2&ibu=3&date=4&left=5&price=6&desc=desc1&regionId=1'

```
Logs:
```
api-gateway_1   | DATA { method: 'POST',
api-gateway_1   |   body: 'abv=1&blg=2&date=4&desc=desc1&ibu=3&left=5&name=name1&price=6&regionId=1&style=style1&userId=1' }
api-gateway_1   | URI http://beer-service:8082/beer/add
api-gateway_1   | RES {"timestamp":"2018-05-06T12:46:03.016+0000","status":400,"error":"Bad Request","message":"Required long parameter 'userId' is not present","path":"/beer/add"}
```

**Comment by Lipsztyk** [comment]
`curl -v -X POST 'http://localhost:8082/beer/add' -d 'name=name1&style=style1&abv=1&blg=2&ibu=3&date=4&left=5&price=6&desc=desc1&regionId=1&userId=1'` worked for me.
Result from `get` in browser `http://localhost:8082/beer/get?id=1` :
`"beerId":1,"userId":1,"name":"name1","style":"style1","abv":1,"blg":2,"ibu":3,"date":4,"left":5,"price":6,"desc":"desc1","region":{"regionId":1,"city":"Warszawa"}}`

**Comment by msypetkowski** [comment]
Yes, directly works. Requests through api-gateway are the problem (http://localhost:3000/api)

**Comment by msypetkowski** [comment]
Fixed in [bb204af](https://github.com/Zwirek009/RSOProject/commit/bb204af9c0fc98921d8d9a1a2830ee1740e96eae)

**Comment by Zwirek009** [comment]
Add sorting data
